### ISSUE TITLE
[Bug][A11y] Private browsing is focused for a second at first launch

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#4447]
## Steps to reproduce
As a follow-up to https://github.com/mozilla-mobile/fenix/issues/2142:
1. Turn TB on.
2. Launch Fenix.
### Expected behavior
The search bar is focused and read.
### Actual behavior
The private browsing button is focused first for a second. 
On Nexus 5 it reads the "Enable" part and then jumps to the search bar. 
On Pixel it doesn't read it but it is selected first.
### Device information

* Android device: Nexus 5 (Android 6.0.1), Pixel 3 (android 9) - using TalkBack
* Not Reproducing on a Galaxy S8 (Android 9) - using Samsung voice assistant
* Fenix version: RC 1.2-rc1

**Comment by ekager** [comment]
I undid the accessibility changes from #2142 so this should be resolved now, closing
